### PR TITLE
Default region is dynamically determined (from hard-coded NZ)

### PR DESCRIPTION
**Pull request by sungeunbae**

### CODE REVIEW

**Review by jpa198 — APPROVED** [review]

File: sources/plot_srf_map.py [hunk 1]
```diff
@@ -346,14 +351,15 @@ def get_args():
 
 ### PART B: NZ map
```
Comment (jasonmotha) [inline sources/plot_srf_map.py:352]: Updating comments to remove reference to NZ would be a good idea in my opinion.

Comment (sungeunbae) [reply sources/plot_srf_map.py]: Updated the comment

File: sources/plot_srf_map.py [hunk 2]
```diff
@@ -215,9 +219,9 @@ def get_args():
 )
 # this is how high the NZ map will end up being
```
Comment (jasonmotha) [inline sources/plot_srf_map.py:220]: This one too. It might be worth - "ctrl+f"ing the file.

Comment (sungeunbae) [reply sources/plot_srf_map.py]: Done

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**